cmd/gsw/axtHelper.go

```diff
@@ -6,18 +6,37 @@ import (
 	"github.com/vertgenlab/gonomics/fileio"
 	"github.com/vertgenlab/gonomics/simpleGraph"
 	"github.com/vertgenlab/gonomics/vcf"
+	"os"
+	"path/filepath"
+	"strings"
+	"sync"
 )
 
 func convertAxt(axtFile, format, targetFa, output string) {
 	switch format {
 	case "vcf":
 		vcfChannel := goChannelAxtVcf(axtFile)
+		tmp := output + ".tmp"
+		file := fileio.EasyCreate(tmp)
```

Comment:
I am not sure what we gain by writing to disk.  Why not read it into a slice from the channel and then sort that slice.  It looks like right now we go from a channel to file, a file to a slice, and then a slice to a sorted slice.  Seems like we can just cut out the file.